Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockRoslynAnalyzerProvider.cs

```diff
@@ -27,17 +27,15 @@
 
 namespace SonarScanner.MSBuild.PreProcessor.Test;
 
-internal class MockRoslynAnalyzerProvider : RoslynAnalyzerProvider
+internal class MockRoslynAnalyzerProvider(BuildSettings buildSettings, IAnalysisPropertyProvider sonarProperties, IEnumerable<SonarRule> rules, string language) : RoslynAnalyzerProvider(new MockAnalyzerInstaller(), new TestLogger(), buildSettings, sonarProperties, rules, language)
```

Comment:
This line is too long and can be reasonably split. We'd use this in normal (long) ctor too
```suggestion
internal class MockRoslynAnalyzerProvider(BuildSettings buildSettings, IAnalysisPropertyProvider sonarProperties, IEnumerable<SonarRule> rules, string language) 
    : RoslynAnalyzerProvider(new MockAnalyzerInstaller(), new TestLogger(), buildSettings, sonarProperties, rules, language)
```